contracts/bonding/BondingManager.sol

```diff
@@ -866,17 +853,18 @@ contract BondingManager is ManagerProxyTarget, IBondingManager {
 
         // Create reward based on active transcoder's stake relative to the total active stake
         // rewardTokens = (current mintable tokens for the round * active transcoder stake) / (total active stake + treasury stake adjustment)
-        uint256 rewardTokens = minter().createReward(
-            earningsPool.totalStake,
-            currentRoundTotalActiveStake + currentRoundTreasuryArtificialStake
-        );
+        uint256 totalRewardTokens = minter().createReward(earningsPool.totalStake, currentRoundTotalActiveStake);
+
+        // The wholesome of treasury rewards were already minted on round initialization, only calculate them to subtract.
+        uint256 treasuryRewards = PreciseMathUtils.percOf(totalRewardTokens, treasuryRewardCutRate);
+        uint256 transcoderRewards = totalRewardTokens.sub(treasuryRewards);
```

Comment:
Might want to invert this calculation (with a similar "cut rate complement") so we dont **over**mint instead of **under**mint